src/content.rs

```diff
@@ -20,15 +20,15 @@ impl Operation {
 }
 
 #[derive(Debug, Clone)]
-pub struct Content {
-	pub operations: Vec<Operation>,
+pub struct Content<Operations: AsRef<[Operation]> = Vec<Operation>> {
```

Comment:
I'm not sure if having this default is desirable, but that should avoid downstream breakages.
If not now, then perhaps in a future version when a breaking change is added anyways, this can be removed if deemed inappropriate.